index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JS Quiz</title>
+</head>
+<body>
+  <label for="height">Height:</label>
+  <input type="number" name="height" id="height" min="0">
```

Comment:
If you wanted to be super technical, the requirements asked for a text input. Not a number input. If you feel like a bit of practice, why not try changing it to a text input and still have it validate that you only entered a number? (I approve of your use of min=0 to prevent negative heights though!)